src/services/auth.ts

```
@@ -207,4 +212,12 @@ export const AuthService = {
 		});
 		return data?.updatePasswordWithEmail ?? '';
 	},
+	withdrawUser: async (client: ApolloClient<any>, emailVerifyTicket: string): Promise<boolean> => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)